docs/general/installation/truenas-scale.md

```diff
@@ -0,0 +1,84 @@
+---
+uid: installation-truenas-scale
+title: TrueNAS SCALE
+description: Install on TrueNAS SCALE.
+sidebar_position: 5
+---
+
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.com/truenas-scale/) community app. The source code for the app is available on the TrueNAS [GitHub](https://github.com/truenas/charts/tree/master/community/jellyfin).
+
+### Notes
+
+    - For setup/install/network questions, try the [TrueNAS Forums](https://forums.truenas.com/).
+        - Seeking help for TrueNAS SCALE on the [Jellyfin Forums](https://forum.jellyfin.org/) will depened on active forum members having experience with SCALE.
+    - TrueNAS SCALE is a NAS OS built on Debian that provides an interface to deploy apps. TrueNAS is provided and maintained by [iXsystems](https://www.ixsystems.com/).
+        - Up until SCALE v24.04 (Dragonfish), Kubernetes was used to deploy apps.
+        - According to [an iXsystems staff member's post](https://forums.truenas.com/t/the-future-of-electric-eel-and-apps/5409), starting with SCALE v24.10 (Electric Eel), Kubernetes will be replaced with Docker to deploy apps on SCALE. You'll be able to use Docker Compose on SCALE. The staff member states for those upgrading their SCALE system to v24.10, any installed community apps (only) will be auto-converted to Docker Compose format. This means custom and 3rd party app deployments on SCALE may need user intervention to reconfigure moving forward. Refer to the container installation guide for further assistance with Docker when the time comes.
+        - The steps outlined below apply to SCALE v24.04.
```

Comment:
Why use a code block for this part?

Reply:
That was unintentional. I was simply visually formatting by eye and wanted a bullet-point list to be seen as belonging to the above "Notes" section. I'm new to markdown, so I wasn't aware a couple of spaces also counted as a code block (just looked it up). I thought backticks or tildes were the only way to make a code block.

You can go ahead and remove the spaces or wait for me to update the pull request later today.